Basic-Video-Chat/app/build.gradle

```diff
@@ -1,31 +1,32 @@
 apply plugin: 'com.android.application'
+apply from: '../../build.properties'
 
 android {
-    compileSdkVersion 28
-    buildToolsVersion '28.0.3'
+    compileSdkVersion android.compileSdkVersion
+    buildToolsVersion android.buildToolsVersion
 
     defaultConfig {
         applicationId "com.tokbox.android.tutorials.basic_video_chat"
         minSdkVersion 16
-        targetSdkVersion 28
-        versionCode 1
-        versionName "1.0"
+        targetSdkVersion targetsdkversion
+        versionCode versioncode
+        versionName versionname
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility sourcecompatibility
```

Comment:
Not sure if this should be configurable. It needs to be 1_8 or it won't build.

I think I will leave JavaVersion.VERSION_1_8 fixed here

Reply:
I checked and it is compiling for all the projects.